### ISSUE TITLE
Installing CRC in a directory other than the home directory

### ISSUE THREAD

**Issue by AlphaMalik** [code-ready/crc#1222]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it: Yes

## CRC version
crc version: 1.9.0+a68b5e0
OpenShift version: 4.3.10 (embedded in binary)
  
## CRC status
Machine 'crc' does not exist. Use 'crc start' to create it.

## CRC config
empty... it's using default values

### Steps to reproduce
On Windows:
  1. Set your home directory to a network directory  (in my case H:)
  2. Run crc setup
  3. Run crc start and follow onscreen instructions

### Expected
A CRC environment is created and ready for use.

### Actual
Command:
C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive Hyper-V\New-VM crc -Path 'H:\.crc\machines\crc' -SwitchName 'Default Switch' -MemoryStartupBytes 8192MB

Fails with this reason:
[stderr =====>] : Hyper-V\New-VM : Creating the virtual machine failed.

When I try this with C: it works but since the home drive on my corporate laptop is set to a network drive, the Hyper-V command fails.

Is there a way to instruct CRC to not load the CRC VM file from the Home drive but retrieve it from a custom location.

### Logs
https://gist.github.com/AlphaMalik/6fe04f439623a3a8de5ab35fab9aad68

**Comment by roldanbob** [comment]
I was trying to do this same thing, because CRC installation failed on two machines (RHEL CSB and Win10 Pro) due to insufficient space in the home directory.  The Windows machine has an SSD as its boot drive, and then two larger partitions on slower disks where I'd like to put CRC. Would love the ability to customize the installation location.

**Comment by gbraad** [comment]
This is even documented as part of the release notes. The binary has to be located on the same disk as the Windows install. This is a shortcoming in the dependent librares we use and has been an issue since minikube/minishift.

**Comment by gbraad** [comment]
Duplicate of #559

**Comment by AlphaMalik** [comment]
If anyone's interested, I got around the problem by first overriding the HOMEDRIVE and HOMEPATH and then running the `crc setup` and `crc start` commands from within the same command window.

**Comment by pmondal08** [comment]
i worked it around by copying the crc.exe to $HOME/

**Comment by cameronkerrnz** [comment]
This is a real problem for those on Windows with a managed device that have their HOME on a remote file server.

This will potentially also be a problem for people that have their home directory synchronised.

In particular, $HOME\.crc\cache\temp-extract is slow, and really should use TEMPDIR or TMPDIR is possible.

In my instance, I have to try and work with the following (summary of environment variables)

```
PS> gci env:

Name                           Value
----                           -----
APPDATA                        C:\Users\ME\AppData\Roaming
HOMEDRIVE                      H:
HOMEPATH                       \
HOMESHARE                      \\mydomain.example.com\path\to\ME
LOCALAPPDATA                   C:\Users\ME\AppData\Local
SystemDrive                    C:
SystemRoot                     C:\WINDOWS
TEMP                           C:\Users\ME\AppData\Local\Temp
TMP                            C:\Users\ME\AppData\Local\Temp
USERPROFILE                    C:\Users\ME
windir                         C:\WINDOWS
```

The setup locations really need to be configurable (and the locations documented)

LOCALAPPDATA may be a better location; that seems to be the place to put cache content (and is on the same volume as SYSTEMDRIVE (at least in my case).

cf. https://www.advancedinstaller.com/appdata-localappdata-programdata.html

**Comment by maxx27** [comment]
> This is a real problem for those on Windows with a managed device that have their HOME on a remote file server.

Yeah, the only workaround I found is just to disconnect this drive and `crc` starts using my `%userprofile%` folder.

BTW, here the issue https://github.com/code-ready/crc/issues/817 to change location.
